### DISCUSSION TITLE
I can't replay any scores

### DISCUSSION THREAD

**Discussion by dustin10weering** [ppy/osu#22833] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Hello, I have had lazer for more than a year now and I have never been able to replay a score, except for those that i just played. After I quit the result screen I can't replay that score anymore.

I'll include a current and old (from 3 july 2022) screenshot. I will also include a video where i was dumb and pressed the wrong replay button, and showing I can't play it again.

I do not know if it is just me that has this issue.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/39807316/224342928-bd9d911d-b032-4ce5-91e1-7ff4deb3e1c8.png)
![osu_2022-07-03_00-40-29](https://user-images.githubusercontent.com/39807316/224344268-c7f99e6b-160c-4ee1-a98c-5c82b2bf198e.jpg)

https://user-images.githubusercontent.com/39807316/224345499-856e0648-f61a-4727-b599-2ace08fcdbcf.mp4

### Version

2023.305.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10943043/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/10943045/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/10943046/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10943047/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10943048/updater.log)
[database.log](https://github.com/ppy/osu/files/10943049/database.log)
[input.log](https://github.com/ppy/osu/files/10943050/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10943051/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10943052/network.log)

**Comment by Theighlin** [comment, marked as answer]
You can replay them from the local leaderboard for now.

**Reply by dustin10weering** [reply]
Thank you, I did not know that :)
